Dockerfile

```diff
@@ -67,25 +67,14 @@ RUN if [ "$TARGETPLATFORM" = "linux/arm64" ] && [ -e "/modulus/deps/vtk-9.2.6.de
     fi
 RUN pip install --no-cache-dir "pyvista>=0.40.1"
 
-# Install DGL from source
+# Install DGL, below instructions only work for containers with CUDA >= 12.1 
+# (https://www.dgl.ai/pages/start.html)
 ARG DGL_BACKEND=pytorch
 ENV DGL_BACKEND=$DGL_BACKEND
 ENV DGLBACKEND=$DGL_BACKEND
-RUN if [ "$TARGETPLATFORM" = "linux/amd64" ] && [ -e "/modulus/deps/dgl-1.1.2-cp310-cp310-linux_x86_64.whl" ]; then \
-        echo "DGL wheel for $TARGETPLATFORM exists, installing!" && \
-        pip install --force-reinstall --no-cache-dir /modulus/deps/dgl-1.1.2-cp310-cp310-linux_x86_64.whl; \
-    elif [ "$TARGETPLATFORM" = "linux/arm64" ] && [ -e "/modulus/deps/dgl-1.1.2-cp310-cp310-linux_aarch64.whl" ]; then \
-        echo "DGL wheel for $TARGETPLATFORM exists, installing!" && \
-        pip install --force-reinstall --no-cache-dir /modulus/deps/dgl-1.1.2-cp310-cp310-linux_aarch64.whl; \
-    else \
-        echo "No DGL wheel present, building from source" && \
-	git clone https://github.com/dmlc/dgl.git && cd dgl/ && git checkout tags/1.1.2 && git submodule update --init --recursive && \
-	DGL_HOME="/workspace/dgl" bash script/build_dgl.sh -g && \
-	cd python && \
-	python setup.py install && \
-	python setup.py build_ext --inplace; \
-    fi
-RUN rm -rf /workspace/dgl
+
+RUN pip install  dgl -f https://data.dgl.ai/wheels/cu121/repo.html
```

Comment:
2 things:
1. I would keep `--no-cache-dir` option to keep Docker image smaller (unless DGL will be re-installed later again).
2. There seems to be 2 spaces in `install  dgl`

Reply:
Good catch on the `--no-cache-dir` option. I added it for other packages, but somehow forgot for this one. Also fixed the spaces.